HISTORY.rst

```diff
@@ -19,7 +19,8 @@ Important Changes:
 
 Other Changes:
 
-* Prevent ``click`` API v7.0 from showing choices when already shown thanks to `@rly`_ (#1168)
+* Test the codebase with python3.8 beta on tox and travis-ci, thanks to `@mihrab34`_ (#1168)
```

Comment:
This should be #1206 right?

Reply:
😱 Good point

Reply:
Fixed on f8470a8c4b01e213074c8514d2710f2f990faa47